content/types/_paginated_result.textile

```diff
@@ -4,16 +4,19 @@ h4.
   default: Properties
   java:    Members
   ruby:    Attributes
+  python:  Attributes
 
-- items := contains a page of results (for example an Array of "@Message@":#message or "@PresenceMessage@":#presence-message objects for a channel history request)<br>__Type: @Array <Message, Presence, Stats>@__
+- items := contains a page of results (for example an Array of "@Message@":#message or "@PresenceMessage@":#presence-message objects for a channel history request)<br>__Type: @List <Message, Presence, Stats>@__
```

Comment:
You've just made this a list for all langs, whereas others generally refer to it as an Array?